### PR TITLE
Added missing Windows constants

### PR DESCRIPTION
**Pull request by kukulich**

### CODE REVIEW

File: Core/Core_d.php [hunk 1]
```diff
@@ -310,6 +310,17 @@
 define ('PHP_WINDOWS_VERSION_SP_MAJOR', 0);
 define ('PHP_WINDOWS_VERSION_SP_MINOR', 0);
 define ('PHP_WINDOWS_VERSION_SUITEMASK', 0);
+define ('PHP_WINDOWS_NT_DOMAIN_CONTROLLER', 0);
```
Comment (andrey-sokolov) [inline Core/Core_d.php:313]:
```suggestion
define ('PHP_WINDOWS_NT_DOMAIN_CONTROLLER', 2);
```

File: Core/Core_d.php [hunk 2]
```diff
@@ -310,6 +310,17 @@
...
+define ('PHP_WINDOWS_NT_SERVER', 0);
```
Comment (andrey-sokolov) [inline Core/Core_d.php:314]:
```suggestion
define ('PHP_WINDOWS_NT_SERVER', 3);
```

File: Core/Core_d.php [hunk 3]
```diff
@@ -310,6 +310,17 @@
...
+define ('PHP_WINDOWS_NT_WORKSTATION', 0);
```
Comment (andrey-sokolov) [inline Core/Core_d.php:315]:
```suggestion
define ('PHP_WINDOWS_NT_WORKSTATION', 1);
```

Comment (kukulich) [reply Core/Core_d.php]: Done

File: Core/Core_d.php (hunk 2 above)
Comment (kukulich) [reply Core/Core_d.php]: Done

File: Core/Core_d.php (hunk 1 above)
Comment (kukulich) [reply Core/Core_d.php]: Done

**Pull request merged**